### PR TITLE
drop day chart updates

### PR DESCRIPTION
**Pull request by sandikbarr**
#384, #391, and #392 
* Release dates are shown in local timezone. (They were already showing in local timezone in the episode select menu, so this just had to change the download/drop day table.)
* Fixes Google analytics, missed a breaking change in the ng upgrade to initialize the tracking
* Drop day table updates:
  * Use smooth lines, chart type: 'spline'
  * Round the line ends (stroke-linecap: round)
  *  Remove bar chart (to be replaced at some point by the 28 day performance feature)

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -27,7 +27,9 @@ export class AppComponent implements OnInit {
   constructor(
     public store: Store<any>,
     private angulartics2GoogleAnalytics: Angulartics2GoogleAnalytics
-  ) {}
+  ) {
```
Comment (sandikbarr) [inline src/app/app.component.ts]: Publish is a version behind metrics on the Google Analytics library. This is the new way to initialize it.

File: src/app/shared/menu/date/standard-date-range-dropdown.component.ts [hunk 2]
```diff
@@ -46,6 +46,6 @@ export class StandardDateRangeDropdownComponent {
   googleAnalyticsEvent(action: string, standardRange: string) {
     const dateRange = dateUtil.getBeginEndDateFromStandardRange(standardRange);
     const value = dateUtil.getAmountOfIntervals(dateRange.beginDate, dateRange.endDate, this.interval);
-    this.store.dispatch(new GoogleAnalyticsEventAction({gaAction: 'filter-' + action, value}));
+    this.store.dispatch(new GoogleAnalyticsEventAction({gaAction: 'routerParams-' + action, value}));
```
Comment (sandikbarr) [inline src/app/shared/menu/date/standard-date-range-dropdown.component.ts]: whoops, other GA events related to the router params were renamed a while back.

File: src/app/shared/util/date/date.format.ts [hunk 3]
```diff
@@ -20,6 +20,14 @@ export const monthDateYear = (date: Date | moment.Moment, separator = true): str
   }
 };
 
+export const monthDateYearLocalTZ = (date: Date | moment.Moment, separator = true): string => {
```
Comment (cavis) [inline src/app/shared/util/date/date.format.ts]: Still a little confusing that the hourly graph shows UTC times.  But hey, getting better!

File: src/app/downloads/chart/dropday-chart-presentation.component.ts [hunk 4]
```diff
@@ -1,18 +1,15 @@
...
   selector: 'metrics-dropday-chart-presentation',
   template: `
-    <prx-category-chart *ngIf="chartData && chartData.length && isBarChart"
-      [data]="chartData" dataLabel="{{routerParams?.days + ' Day Downloads'}}">
-    </prx-category-chart>
-    <prx-indexed-chart *ngIf="chartData && chartData.length && isLineChart" type="line" [datasets]="chartData" [type]="chartType">
-    </prx-indexed-chart>
-    <p *ngIf="chartData && chartData.length && isLineChart">{{intervalLabel}} since drop</p>
+    <ng-container *ngIf="chartData && chartData.length">
+      <prx-indexed-chart type="spline" [datasets]="chartData" [formatY]="formatY"></prx-indexed-chart>
```
Comment (cavis) [inline src/app/downloads/chart/dropday-chart-presentation.component.ts]: Spline++

File: src/app/app.component.ts (hunk 1 above)
Comment (cavis) [reply src/app/app.component.ts]: +1

File: src/app/shared/util/date/date.format.ts (hunk 3 above)
Comment (sandikbarr) [reply src/app/shared/util/date/date.format.ts]: It's actually showing local timezone with hourly data in the chart and in the data for export, or did I miss that somewhere? The date selectors are still UTC based when hourly is selected. It is a weird mix.

**Pull request merged**